windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -31,23 +31,36 @@ endif::openshift-origin[]
 [id="wmco-6-0-0"]
 == Release notes for Red Hat Windows Machine Config Operator 6.0.0
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in https://access.redhat.com/errata/RHBA-[2022:97930].
```

Comment:
Broken link

Reply:
Additional clarification - should be formatted like:

```
link:https://access.redhat.com/errata/RHBA-2022:1234[RHBA-2022:1234]
```

The `RHBA-2022:97930` in the current text appears to be an internal temporary ID (5 digits), rather than a Live ID (they only have 4 digits). You will need the Live ID for the actual user docs.

Reply:
@jeana-redhat correct me if im wrong , the live ID is generated when the errata goes live which means that I would need to update the link on 23rd Aug (GA).

Reply:
Hey @Amrita42 - nope, assigning a Live ID is a manual step and can be done at any time after an advisory is created. Typically this is done by the person who approves the docs page of the advisory.

The link as you have it here will not work - it's formatted like a Live ID but with the temp ID number. The temp ID link would be https://errata.devel.redhat.com/advisory/97930, but that link requires an employee login :slightly_smiling_face: